src/chatview/chatView.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styles from "./styles";
+import { withStyles } from "@material-ui/core/styles";
+
+class ChatViewComponent extends React.Component {
+  componentDidUpdate = () => {
+    const container = document.getElementById("chatview-container");
+
+    if (container) {
+      container.scrollTo(0, container.scrollHeight);
+    }
+  };
+
+  render() {
+    const { classes, chat, user } = this.props;
+
+    if (chat === undefined) {
```

Comment:
You could write this as `if (!chat) {` and it would do the exact same thing. As you probably know the exclamation mark inverts the operator, so false becomes true.

Reply:
Thanks, Nick, I did as you said.